src/test/java/co/rsk/federate/config/PowpegConfigBuilder.java

```diff
@@ -0,0 +1,41 @@
+package co.rsk.federate.config;
+
+import static com.typesafe.config.ConfigValueFactory.fromAnyRef;
+
+import com.typesafe.config.Config;
+import com.typesafe.config.ConfigFactory;
+
+public class PowpegConfigBuilder {
```

Comment:
Goind through this again, looks like this builder is only building the config for different signers. Is that right?

If that's the case we should rename it, `SignerConfigBuilder` or something of the sort.

And should this return a `Config` object or the `SignerConfig` object directly? They `key` could be passed to the builder constructor or to the `build` method. What do you think?